winui/Masked-TextBox/Error-Indication.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: Error Indication in WinUI MaskedTextBox control | Syncfusion
+description: Explore the various error types supported by the Syncfusion WinUI MaskedTextBox (SfMaskedTextBox) control to enhance user experience and input validation.
+platform: WinUI
+control: SfMaskedTextBox
+documentation: ug
+---
+
+# Error Indication in WinUI MaskedTextBox
+
+The [WinUI MaskedTextBox](https://www.syncfusion.com/winui-controls/masked-textbox) control provides comprehensive error types support to facilitate effective input validation and enhance user experience. This feature enables users to utilize various error icons and styles based on specific validation scenarios.The following error types such as none, default, warning, information, critical, success, custom is available in the MaskedTextBox control.
+
+## Built-in Error types:
+
+* `None` : No visible signals for errors.
+* `Default` : Red bottom border brush in the MaskedTextBox Control.
+* `Warning ` : Visual highlight for potential concerns needing immediate attention.
+* `Information` : Share non-essential details or guidance.
+* `Critical` : Urgent issues requiring immediate action.
+* `Success` : Positive confirmation for valid input or successful actions.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfMaskedTextBox Name="maskedTextBox"
+                            MaskType="RegEx" 
+                            Mask="[a-z0-9._%-]+@[a-z0-9]+\.[a-z]{2,3}" 
+                            Width="200"
+                            ErrorType="Warning"
+                            ValueChanging="errorTypeMaskTextBox_ValueChanging"/>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfMaskedTextBox maskedTextBox = new SfMaskedTextBox();
+maskedTextBox.MaskType = MaskedTextBoxMaskType.RegEx;
+maskedTextBox.Mask = "[a-z0-9._%-]+@[a-z0-9]+\\.[a-z]{2,3}";
+maskedTextBox.Width = 200;
+maskedTextBox.ErrorType = ErrorType.Warning;
+maskedTextBox.ValueChanging += errorTypeMaskTextBox_ValueChanging;
+
+// If the entered value is valid then success icon will display in the SfMaskedTextBox .
+
+private void errorTypeMaskTextBox_ValueChanging(object sender, MaskedTextBoxValueChangingEventArgs e)
+{
+    maskedTextBox.ErrorType = e.IsValid ? ErrorType.Success : ErrorType.Warning;
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI MaskedTextBox control with Errortype](MaskedTextBox_images/winui_masked_textbox_errortype.png)
+
+## Custom:
+
+The `Custom` error type in the SfMaskedTextBox allows users to define a unique error icon and customize the error border color using the `CustomErrorIcon` and `CustomErrorBorderBrush` properties, respectively.
+
+{% tabs %}
```

Comment:
@GilbertmanojSF3971 
## Custom error type:
The `Custom` error type provides users with the flexibility to customize both the error icon and the border brush. This customization is achieved through the following properties:

CustomErrorIcon: Allows you to set a custom error icon.
CustomErrorBorderBrush: Enables customization of the border brush for the `Custom` error type.

Reply:
Hi @Manivannan-E Sir , Modified the description content for the Custom error type.